tests/trunc-double-varying.ispc

```diff
@@ -0,0 +1,78 @@
+
+export void f_f(uniform float RET[], uniform float aFOO[]) {
+
+    float error = 0;
+    RET[programIndex] = 0;
+    double deltaLow = 0.000001d;
+    double deltaHigh = 0.999999d;
+    double testVal;
+    double resVal;
+    double baseVal;
+    double checkVal = 0;
+
+    //Case 1 : Covers range  -(programCount/2 -1) < val < +(programCount/2) including -1 < val < +1 with deltaLow.
+    // So, this covers +0.0, -0.0, -inf < val < 0 with decimal greater than 0.5, 0 < val < +inf with decimal less than 0.5
+    //For eg. for a 8 wide target, this will check values : -2.999999,-1.999999,-0.999999,0.000001,1.000001,2.000001,3.000001,4.000001
+    baseVal = aFOO[programIndex] - (programCount / 2);
+    testVal = baseVal + deltaLow;
+    resVal = trunc(testVal);
+    checkVal = baseVal;
+    if (baseVal < 0) {
+        checkVal = -abs(baseVal + 1); // -abs() Needed for -0.0
+    }
+    if (checkVal != resVal) {
+        error++;
+    }
+
+    //Case 2 : Covers range  -(programCount/2 -1) < val < +(programCount/2) including -1 < val < +1 with deltaHigh.
+     // So, this covers +0.0, -0.0, -inf < val < 0 with decimal less than 0.5, 0 < val < +inf with decimal greater than 0.5
```

Comment:
Identation.

Reply:
Done.

Reply:
This fixed in other tests, but not in this one.